[instruments] updated README and test plan for widgets (#6484)
aces/Loris: modules/instruments/test/TestPlan.md

12. Verify Validity flags are functioning for instruments.
    - **Validity:** can be marked as “Valid”, “Questionable”, or “Invalid”. Whether or not this flag is shown for an instrument is by the boolean $ValidityEnabled. Whether the field is required before flagging an instrument as complete is determined by $ValidityRequired.
    - You can test the forgoing flags by switching the corresponding boolean of the test instruments to either `true` or `false`.

> 13. Dashboard Widget - "My Tasks" for Incomplete forms

Reviewer: Can you put these in different sections? It's not clear the way it's written that the user needs to go to a different module to validate these things.

Author: @driusan done

12. Verify Validity flags are functioning for instruments.
    - **Validity:** can be marked as “Valid”, “Questionable”, or “Invalid”. Whether or not this flag is shown for an instrument is by the boolean $ValidityEnabled. Whether the field is required before flagging an instrument as complete is determined by $ValidityRequired.
    - You can test the forgoing flags by switching the corresponding boolean of the test instruments to either `true` or `false`.
13. Dashboard Widget - "My Tasks" for Incomplete forms
    - Ensure the total of Incomplete forms correspond with the correct 
        number of candidates inside the statistics/statistics_site module.
    - Click on the total number of Incomplete forms and check if redirection
        to the statistics/statistics_site module succeeds.
14. Candidate Widget - "Behavioural Data" for Candidate instruments entry.
    - Ensure the appropriate visit labels are visible for the candidate. 
    - The visit status, subproject status, site, date of visit, and age should be shown.
    - Clicking on a visit should redirect to the instrument_list module for the candidate.

>     - Test making a new Time Point for a candidate and assign a Visit label. The Behavioural Data widget should update 

Reviewer 2: ```suggestion
    - Test making a new Time Point for a candidate and assign an instrument to the time point. The Behavioural Data widget should update accordingly.
```

>         to include the new Time Point by showing the visit label with the subproject, site, and status as not started.

Reviewer 2: ```suggestion
        to include the new Time Point by showing the instrument with the subproject, site, and status as not started.
```

Reviewer 2: I made the 2 previous comment with a length of time in between. This line is now superfluous.

12. Verify Validity flags are functioning for instruments.
    - **Validity:** can be marked as “Valid”, “Questionable”, or “Invalid”. Whether or not this flag is shown for an instrument is by the boolean $ValidityEnabled. Whether the field is required before flagging an instrument as complete is determined by $ValidityRequired.
    - You can test the forgoing flags by switching the corresponding boolean of the test instruments to either `true` or `false`.

### The Dashboard module corresponding to Instruments module.

>    1. Dashboard Widget - "My Tasks" for Incomplete forms 

Reviewer: This should tell people they need to go to the dashboard to do this test.

Author: @driusan fixed

Reviewer: I don't see the fix.. did you forget to push it?

      - Ensure the total of Incomplete forms correspond with the correct 
            number of candidates inside the statistics/statistics_site module.
      - Click on the total number of Incomplete forms and check if redirection
            to the statistics/statistics_site module succeeds.

### The Candidate_list module corresponding to Instruments module.

>    1. Candidate Widget - "Behavioural Data" for Candidate instruments entry.

Reviewer: This should tell people that they need to go to the candidate dashboard for a candidate.

Author: @driusan fixed.
